aten/src/ATen/native/GridSampler.cpp

```diff
@@ -471,11 +354,6 @@ namespace {
                   }
                 }
 
-                // un-normalize grad_grid values back to [-1, 1] constraints
-                gix = gix * (inp_W - 1) / 2;
-                giy = giy * (inp_H - 1) / 2;
-                giz = giz * (inp_D - 1) / 2;
```

Comment:
Note that this is factored out into the `gi[x,y,z]_mult` produced by calling the `grid_sampler_compute_source_index_set_grad()` function above.

(Just in case someone is confused by this removal.)